README.md

```diff
@@ -152,19 +163,29 @@ gcloud init
 gcloud auth application-default login
 ```
 
-You will generate a file used for authentication of Google cloud api in jigasi. You will see a result like:
+You will generate a file used for authentication of Google cloud api in Jigasi. You will see a result like:
 `Credentials saved to file: [/root/.config/gcloud/application_default_credentials.json]`
-Move the file to jigasi config and change its permissions:
+
+Move the file to Jigasi config and change its permissions:
+
 ```
 mv /root/.config/gcloud/application_default_credentials.json /etc/jitsi/jigasi
 chown jigasi:jitsi /etc/jitsi/jigasi/application_default_credentials.json
 ```
+
 In the file `/etc/jitsi/jigasi/config` add at the end:
+
 ```
 # Credential for Google Cloud Speech API
 GOOGLE_APPLICATION_CREDENTIALS=/etc/jitsi/jigasi/application_default_credentials.json
 ```
-Restart jigasi
+
+Set the following property in `~/jigasi/jigasi-home/sip-communicator.properties`:
```

Comment:
Why this file in the docs in rest of the places we refer to `/etc/jitsi/jigasi/sip-communicator.properties`?